src/util/sawyercoding.c

```diff
@@ -429,16 +431,17 @@ static size_t encode_chunk_repeat(const uint8 *src_buffer, uint8 *dst_buffer, si
 
 	// Iterate through remainder of the source buffer
 	for (i = 1; i < length; ) {
-		searchIndex = max(0, i - 32);
+		searchIndex = (i < 32) ? 0 : (i - 32);
```

Comment:
`i` and`searchIndex` are `size_t`, so the `max` here wasn't the proper way to check.